free-tunes/Controllers/TrackExplorerViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CategoriesViewController.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/02/28.
+//
+
+import UIKit
+import AVFoundation
+
+class TrackExplorerViewController: UIViewController {
+    
+// MARK: - Interface Builder Outlets
+    @IBOutlet private weak var trackCollectionView: UICollectionView!
+    
+    private lazy var exploreViewModel = TrackExplorerViewModel()
+    var looper: AVPlayerLooper!
+    var player: AVQueuePlayer!
+
+// MARK: - Runtime Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        trackCollectionView.register(UINib(nibName: "TrackCollectionViewCell", bundle: .main), forCellWithReuseIdentifier: "TrackCollectionViewCell")
+        trackCollectionView.isPagingEnabled = true
+        getExploreViewContent()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        self.navigationController?.setNavigationBarHidden(true, animated: animated)
+        super.viewWillAppear(animated)
+    }
+    
+// MARK: - Controller Logic Methods
+    func setUpExploreCollectionView() {
+        trackCollectionView.dataSource = self
+        trackCollectionView.delegate = self
+    }
+    
+    func getExploreViewContent() {
+        let url = Constants.artistExploreContentUrl
```

Comment:
Should this not be guard  let url = Constants.artistExploreContentUrl see { return } as the url may not exist so it should be guarded

Reply:
the case where the url doesn't exist is handled in the make request method already